cloudyweb/urls.py

```diff
@@ -21,8 +21,8 @@
 
 router = routers.DefaultRouter()
 router.register(r'users', accounts_views.UserViewSet)
-router.register(r'games', cloudygames_views.GameViewSet, 'owned')
-router.register(r'game-session', cloudygames_views.GameSessionViewSet)
+router.register(r'games', cloudygames_views.GameViewSet, base_name='Game')
```

Comment:
Quoted from Django REST Framework - Routers:
Typically you won't need to specify the base_name argument, but if you have a viewset where you've defined a custom get_queryset method, then the viewset may not have a .queryset attribute set.

That's why I need to explicitly specify the base_name.